### PR TITLE
Automate stripe payment

### PR DESCRIPTION
**Pull request by eduardhabryd**

### CODE REVIEW

File: stripe_pay/stripe_utils.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def stripe_payment(data):
+    print(data.__dict__)
```
Comment (Barsh4ec) [inline stripe_pay/stripe_utils.py:2]: I'm not sure if this function is required, but looks vierd

File: stripe_pay/tests.py [hunk 2]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```
Comment (Barsh4ec) [inline stripe_pay/tests.py:3]: remove empty folder

File: stripe_pay/views.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+from payments.models import Payment
+
+
+class PayView(TemplateView):
+    template_name = "pay.html"
+
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        borrowing_id = self.kwargs.get('borrowing_id')
+        context['borrowing_id'] = borrowing_id
```
Comment (Barsh4ec) [inline stripe_pay/views.py:18]: double quotes here

File: stripe_pay/views.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+                line_items=[
+                    {
+                        'price_data': {
+                            'currency': 'usd',
+                            'product_data': {
+                                'name': book_name,
+                            },
+                            'unit_amount': price,
+                        },
+                        'quantity': 1,
```
Comment (Barsh4ec) [inline stripe_pay/views.py:56]: double quotes here

**Review by Barsh4ec — CHANGES_REQUESTED** [review]

**Review by Barsh4ec — APPROVED** [review]

**Review by ZaichenkoViktoriia — APPROVED** [review]

**Pull request merged**